docs/data-management.md

```diff
@@ -135,7 +135,7 @@ CLEAN FILES FOR TABLE table1
    
 | Parameter | Default | Application | Description | Valid Values |
 |-----------------------------------------|---------|-------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------|
-| carbon.compaction.level.threshold | 4, 3 | Minor | This property is for minor compaction which decides how many segments to be merged. Example: If it is set as 2, 3 then minor compaction will be triggered for every 2 segments. 3 is the number of level 1 compacted segment which is further compacted to new segment. | NA |
+| carbon.compaction.level.threshold | 4, 3 | Minor | This property is for minor compaction which decides how many segments to be merged. Example: If it is set as 2, 3, then minor compaction will be triggered for every 2 segments in level 1. 3 is the number of level 1 compacted segment which is further compacted to new segment in level 2. | NA |
```

Comment:
suggest to change `2, 3` to `"2, 3"`

Reply:
suggest to explain level 1 compaction and level 2 compaction first.

Reply:
I just have changed 2, 3 to "2, 3".

" Minor Compaction" part has explained  level 1 compaction and  and level 2 compaction